ostelco-ios-client/ModelControllers/UserDefaultsWrapper.swift

```diff
@@ -57,4 +58,17 @@ struct UserDefaultsWrapper {
             }
         }
     }
+    
+    static var userLoggedOut: Bool {
```

Comment:
I think this is not necessary - if we don't have anything stored for the user in the keychain, we'll have to kick them  to the login flow anyway to get a new JWT, regardless of whether we manage to invalidate their session with Auth0.